ci-operator/step-registry/baremetalds/assisted/operator/ztp/baremetalds-assisted-operator-ztp-commands.sh

```diff
@@ -12,32 +12,11 @@ source "${SHARED_DIR}/packet-conf.sh"
 
 tar -czf - . | ssh "${SSHOPTS[@]}" "root@${IP}" "cat > /root/assisted-service.tar.gz"
 
-# Copy additional dev-script variables, if present
-if [[ -e "${SHARED_DIR}/ds-vars.conf" ]]
-then
-  scp "${SSHOPTS[@]}" "${SHARED_DIR}/ds-vars.conf" "root@${IP}:ds-vars.conf"
-fi
-
-# Copy job variables to the packet server
-echo "export OPENSHIFT_INSTALL_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE}" >> /tmp/assisted-vars.conf
-echo "export ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE=${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE}" >> /tmp/assisted-vars.conf
-echo "export ASSISTED_NAMESPACE=${ASSISTED_NAMESPACE}" >> /tmp/assisted-vars.conf
-echo "export ASSISTED_CLUSTER_NAME=${ASSISTED_CLUSTER_NAME}" >> /tmp/assisted-vars.conf
-echo "export ASSISTED_CLUSTER_DEPLOYMENT_NAME=${ASSISTED_CLUSTER_DEPLOYMENT_NAME}" >> /tmp/assisted-vars.conf
-echo "export ASSISTED_INFRAENV_NAME=${ASSISTED_INFRAENV_NAME}" >> /tmp/assisted-vars.conf
-echo "export ASSISTED_PRIVATEKEY_NAME=${ASSISTED_PRIVATEKEY_NAME}" >> /tmp/assisted-vars.conf
-echo "export ASSISTED_PULLSECRET_NAME=${ASSISTED_PULLSECRET_NAME}" >> /tmp/assisted-vars.conf
-echo "export ASSISTED_PULLSECRET_JSON=${ASSISTED_PULLSECRET_JSON}" >> /tmp/assisted-vars.conf
-scp "${SSHOPTS[@]}" "/tmp/assisted-vars.conf" "root@${IP}:assisted-vars.conf"
```

Comment:
You get unbound variable errors on the packet server without copying the variable files over.

Reply:
They aren't needed or used in the ZTP workflow. 
The configuration happens part of the `operator-setup` step and they are being used with `source /root/config`

Reply:
They are used in the ZTP workflow to configure the spoke cluster. I would argue that the fact that the default values I originally picked caused confusion and uncovered some bugs is a reason to keep them exposed and not obfuscated. We need to have the ability to override and change things. I think instead to clean it up we should move to a configuration file and expose that.

Reply:
I will also point out that those default values are what was floating around our limited documentation at the time. So there has been value in exposing these, even if the reaction is one that "causes confusion."

Reply:
@lranjbar Those variables could be overriden, but the CI shouldn't expose it while it's not actually being used by any workflow. It would just cause those variables to be maintained in two places. Hence, I'd prefer to use the default ones.

https://github.com/openshift/assisted-service/blob/6ef5bdf85aa856c1b8d06d9ed483036f8bb0b4cf/deploy/operator/ztp/deploy_spoke_cluster.sh#L8-L16

I agree about the documentation and about grouping them together so they could be easily changed by a new contributor. But the CI doesn't support it in the short term. e.g. `CLUSTER_NAME` and `DEPLOYMENT_NAME` don't really change anything about the workflow. The only important variable that should be changed by the CI job is `ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE` which is being overidden in the operator-setup step.
https://github.com/openshift/release/pull/19988/files#diff-ada4d7284feee37cf539cece69196ee93dee7cabb1f81f9891a9daab5e8e6c85R48